integration-tests/tests/chain_generic_tests.rs

```diff
@@ -33,6 +34,10 @@ async fn libtonode_propose_and_broadcast_40_000_to_sapling() {
 async fn libtonode_propose_and_broadcast_40_000_to_orchard() {
     propose_and_broadcast_value_to_pool::<LibtonodeEnvironment>(40_000, Shielded(Orchard)).await;
 }
+#[tokio::test]
```

Comment:
im very happy to have spontaneously demonstrated that... the changes made in this branch change this test fixture from FAIL to PASS

Reply:
I think it will be good when the assertions associated with a test are a separate concern from the thing called "fixture".